packages/plugin-ua-parser-browser/src/typings/ua-parser-plugin.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import { EnrichmentPlugin } from '@amplitude/analytics-types';
+
+export interface Options {
+  osName?: boolean;
+  osVersion?: boolean;
+  deviceManufacturer?: boolean;
+  deviceModel?: boolean;
+}
+
+export interface CreateUaParserPlugin {
```

Comment:
```suggestion
export interface EnrichUserAgentPlugin {
```

Reply:
```suggestion
export interface UserAgentEnrichmentPlugin {
```

Reply:
use `CreateEnrichUserAgentPlugin` to align the names with other plugins.

Reply:
`CreateUserAgentEnrichmentPlugin`?